README.md

```diff
@@ -22,6 +22,8 @@ The Firefox Accounts (fxa) monorepo
    npm start
    ```
 
+Note this starts up all required services, including Redis, MySQL, and Memcached. You should not be running them yourself, or occupying any of the [server ports](https://github.com/mozilla/fxa/blob/master/mysql_servers.json). Doing so may result in errors.
```

Comment:
> You should not be running them yourself

nit: I would probably say something like it is recommended that you don't run these services yourself.